slices/libnl-cli-3-200.yaml

```
@@ -0,0 +1,29 @@
+package: libnl-cli-3-200
+
+essential:
+  - libnl-cli-3-200_copyright
+
+slices:
+  cls:
+    essential:
+      - libc6_libs
```

Are we missing libnl-genl-3-200

```
Depends: libnl-3-200 (= 3.7.0-0.3build1.1), libnl-genl-3-200 (= 3.7.0-0.3build1.1), libnl-nf-3-200 (= 3.7.0-0.3build1.1), libnl-route-3-200 (= 3.7.0-0.3build1.1), libc6 (>= 2.38)
```

```suggestion
      - libc6_libs
      - libnl-genl-3-200
```

---

Actually in #609 we found that libnl-cli-3-200 doesn't depends on libnl-genl-3-200: https://github.com/canonical/chisel-releases/pull/609/files/80023581133239cc3e9ba474a1f05e3b61d6989f#r2269472059. The situations are the same for 24.04 and 25.04